src/index.html

```diff
@@ -22,7 +22,7 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+<html spellcheck="false">
```

Comment:
This is a CEF-only issue, so it doesn't apply to other configurations (e.g. in-browser). I also would rather not clutter the HTML markup.